Bukkit/src/main/java/com/plotsquared/bukkit/BukkitPlatform.java

```diff
@@ -999,6 +1003,20 @@ private void runEntityTask() {
         }), TaskTime.seconds(1L));
     }
 
+    private void removeRoadEntity(Entity entity) {
+        RemoveRoadEntityEvent event = eventDispatcher.callRemoveRoadEntity(
+                BukkitAdapter.adapt(entity),
+                BukkitAdapter.adapt(entity.getType()),
+                entity.getUniqueId(),
+                entity.getEntityId()
+        );
+
+        if (Objects.equals(event.getEventResult(), Result.DENY))
```

Comment:
`==` is fine here as we're comparing enums.

Reply:
Since the EventResult can also be null, I use 'Objects#equals' here to prevent a NPE and to perform a null check through it

Reply:
that's should be no issue tho - if the result is no the check would simplify to `null == DENY` -> `false`, which would be the expected behaviour.

Reply:
Ah okay. I don't use `==` normally, think it doesn't really look nice. But sounds logical, I'll change it.